tools/operator/testclient/operations/instantiatechaincode.go

```diff
@@ -208,15 +237,14 @@ func (i InstantiateCCUIObject) getEndorsementPolicy(organizations []inputStructs
 	return endorsementPolicy, nil
 }
 
-//GetMSPIDForOrg -- To get the MSP ID for an organization
-func GetMSPIDForOrg(connProfilePath, orgName string) (string, error) {
+//GetConnProfileInformationForOrg -- To get the MSP ID for an organization
+func GetConnProfileInformationForOrg(connProfilePath, orgName string) (GetConnProfileOptions, error) {
 
-	var config GetMSPID
-	var mspID string
+	var config GetConnProfileOptions
 	if !(strings.HasSuffix(connProfilePath, "yaml") || strings.HasSuffix(connProfilePath, "yml")) {
 		files, err := ioutil.ReadDir(connProfilePath)
 		if err != nil {
-			return "", err
+			return config, err
```

Comment:
If it's an object we should either return `nil` or an empty instance of the object `GetConnProfileOptions{}` explicitly if we can't return nil.

Can we also rename `GetConnProfileOptions` to `ConnProfileOptions`